ref/test/CMakeLists.txt

```diff
@@ -75,3 +82,8 @@ add_test(NAME regression_48x48
 add_test(NAME compare_48x48
          COMMAND ${COMPARE_SH} test_output/c_sw_48x48.log.out test_output/c_sw_48x48.test)
 
+# IsEqual Tests
+add_test(NAME IsEqualTest 
+         COMMAND EqualTest)
```

Comment:
@christopherwharrop-noaa This is where the CI check is failing for the two CI builds. When I run `ctest` locally within the test folder after building the kernel, all of the tests pass. I'm not sure how I can get more information as to why exactly the test is failing in the Github Actions Workflow. But I'll be looking more into this tomorrow, following the [add_test()](https://cmake.org/cmake/help/v3.7/command/add_test.html) docs

Reply:
@robgonzalezpita - Your cmake code is fine.  The problem is that your Fortran code is crashing with a floating point exception for the debug mode build.  When you ran your `ctest` locally, did you try both release and debug builds?

I am seeing:

```
$ ctest -VV -R IsEqualTest

[irrelevant output removed]

9:  ______________________________________________________________________________________________________________________
9:  Tests for Comparisons involving zero
9:  T a=   0.0000000000000000      b=   0.0000000000000000      tol=,     9.9999999999999995E-007
9:  T a=   0.0000000000000000      b=  -0.0000000000000000      tol=,     9.9999999999999995E-007
9:  T a=  -0.0000000000000000      b=   0.0000000000000000      tol=,     9.9999999999999995E-007
9: 
9: Program received signal SIGFPE: Floating-point exception - erroneous arithmetic operation.
9: 
9: Backtrace for this error:
9: #0  0x7feb5524e3ff in ???
9: #1  0x400afe in __isequalmodule_MOD_isequal
9: 	at [removed path information] /ref/test/utilityfunctions/isEqual.f90:26

[irrelevant output removed]

```
We may need to tweak this algorithm a bit.

Reply:
I completely neglected to run it in debug build. I'll start with that, and look into tweaking the algorithm.

Reply:
There are compiler flags that sell the compiler how to set the fp exceptions.  Sometimes INF is a good idea.  Some codes even use NaNs, they are OK, not exceptions.  HUGE+HUGE is INF.  Anyway, check out Intel's -fpe0 and -fpe1 flags.